gcloud-java-core/src/main/java/com/google/gcloud/BaseServiceException.java

```diff
@@ -123,7 +138,7 @@ public BaseServiceException(GoogleJsonError error, boolean idempotent) {
     this.code = error.getCode();
```

Comment:
I wonder if this one could call `this(error.getCode(), error.getMessage(), reason(error), idempotent)` instead?